src/land/land.rs

```diff
@@ -0,0 +1,72 @@
+use bevy::{
+    prelude::*,
+    render::mesh::VertexAttribute,
+    prelude};
+
+
+
+//takes a grayscale texture handle and returns a mesh with height based on the grayscale values
+pub fn texture_to_mesh (
+    textures: ResMut<Assets<Texture>>,
+    land_texture_handle: prelude::Handle<prelude::Texture>,
+) -> bevy::prelude::Mesh {
+
+    //gets a reference to the data needed
+    let land_texture = textures.get(&land_texture_handle).unwrap();
```

Comment:
Use `expect("error message")` in place of `unwrap`. Even better, make the function return `Result<bevy::prelude::Mesh, SomeErrorType>` and don't panic at all.